Documentation/install.rst

```diff
@@ -1097,30 +1097,41 @@ If for some reason you do not want to run Cilium as a container image.
 Installing it from source is possible as well. It does come with additional
 dependencies described in :ref:`admin_system_reqs`.
 
+0. Download and install go-bindata
+
+    $ go get -u github.com/jteeuwen/go-bindata/...
+
 1. Download & extract the latest Cilium release from the ReleasesPage_
 
 .. _ReleasesPage: https://github.com/cilium/cilium/releases
 
 .. code:: bash
 
-    $ wget https://github.com/cilium/cilium/archive/v0.10.0.tar.gz
-    $ tar xzvf v0.10.0.tar.gz
-    $ cd cilium-0.10.0
+   $ go get github.com/cilium/cilium
+   $ cd $GOPATH/src/github.com/cilium/cilium
 
 2. Build & install the Cilium binaries to ``bindir``
 
 .. code:: bash
 
-   $ make
+   $ git checkout v0.11
+   $ # We are pointing to $GOPATH/bin as well since it's where go-bindata is
+   $ # installed
+   $ PATH=$GOPATH/bin:$PATH make
    $ sudo make install
 
-3. Optional: Install systemd/upstart init files:
+3. Optional: Install upstart/systemd init files:
 
 .. code:: bash
 
     sudo cp contrib/upstart/* /etc/init/
```

Comment:
Is there a way we can put this into a script instead of having code in the documentation? That way we can maybe have a test or something in the repository called "doc-script-tests" that tests whether the code used in our documentation still works. I don't think this blocks this from being merged, but is worthy of an enhancement ticket. @scanf what do you think?

Reply:
`literalinclude` should work fine here. It has language options IIRC. Created an issue for this #1901, please add to it if something was properly worded. Thanks.

Reply:
Looks good, thanks for filing!